### PR TITLE
Updates Mapping in Readme and info on the development channel on the steam group.

### PR DESCRIPTION
**Pull request by alao3alao3**
Updates Mapping section by mentioning the other skeleton brush entity (i.e func_skeletongate), how to make a weapon not respawn, where to define gamemode variables for their maps (like barricade health).

Also includes a link to how to reach the channel where people who want to contribute to fatkid can get help and support regarding questions they can reasonably have on code or mapmaking. I dont know how we went this long without noticing that not there.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,23 +8,23 @@ I originally created this gamemode in 2015 and it was [popular on YouTube](https
...
 
-I hope for community contribution to the gamemode. If you make a new map or other extension for it, please be aware that the [license](https://github.com/swampservers/fatkid/blob/master/LICENSE) gives us permission to include your work in this repository so everyone can use it. Extensions are required to be open source as spelled out in the license. We would also appreciate it if VMFs of maps are made available.
+I hope for community contribution to the gamemode. If you make a new map or other extension for it, please be aware that the [license](https://github.com/swampservers/fatkid/blob/master/LICENSE) gives us permission to include your work in this repository so everyone can use it. Extensions are required to be open source as spelled out in the license. We would also appreciate it if VMFs of maps are made available. For further information or help regarding development visit the development channel in the [Swamp Server's group chat room](https://steamcommunity.com/groups/swampservers).
```
Comment (swampservers) [inline README.md]: Swamp Servers, not Swamp Server's

File: README.md [hunk 2]
```diff
@@ -8,23 +8,23 @@ I originally created this gamemode in 2015 and it was [popular on YouTube](https
...
 - Spawnpoints: info_player_start for humans, info_zombie_start for skeletons (if none they use human spawns), info_az_start for fat kid (if none he uses skeleton spawns or human spawns)
-- Respawning weapons: Just place frozen weapon entities; the respawning is handled by code in the gamemode.
-- Map-specific Lua (such as dodgeball spawning and traffic cars): Add your code to a folder with the map name as shown [here](https://github.com/swampservers/fatkid/tree/master/gamemodes/fatkid/gamemode/maps)
+- Respawning weapons: Just place frozen weapon entities; the respawning is handled by code in the gamemode. If you do not want a weapon to respawn, name it "norespawn".
+- Map-specific Lua (such as dodgeball spawning and traffic cars): Add your code to a folder with the map name as shown [here](https://github.com/swampservers/fatkid/tree/master/gamemodes/fatkid/gamemode/maps). Local variables such as barricade health can be defined for your map in it's respective init.lua file.
```
Comment (swampservers) [inline README.md]: "Local variables" actually means something different (like `local x = 1`). A better word for what you're setting might be "gamemode variables"

**Comment by swampservers** [comment]
Thank you, some comments added

**Comment by alao3alao3** [comment]
should be fixed now and ready for merge

**Pull request merged**